packages/core/parcel-bundler/src/assets/TypeScriptAsset.js

```diff
@@ -24,13 +25,30 @@ class TypeScriptAsset extends Asset {
       fileName: this.relativeName
     };
 
-    let tsconfig = await this.getConfig(['tsconfig.json']);
+    // Resolve and parse tsconfig.json file
+    const tsconfigCompilerOptions = (() => {
+      const configFilename = typescript.findConfigFile(
+        this.name,
+        typescript.sys.fileExists
```

Comment:
Is this considered protected by semver by the typescript team?
Seems like an internal function to me.

Don't want to have bug reports about breaking changes on minor releases.

Reply:
Hmm, very good point. I did some research, and I have very little evidence that this could last in the future TypeScript releases. In fact, it is said that it might change [here](https://github.com/Microsoft/TypeScript/wiki/Using-the-Compiler-API). I will rewrite this part to reflect the same functionality without relying on TypeScript API.